scripts/samconfig-zip.toml

```diff
@@ -0,0 +1,27 @@
+# More information about the configuration file can be found here:
+# https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-config.html
+version = 0.1
+
+[default.global.parameters]
+stack_name = "{{cookiecutter.project_name}}"
```

Comment:
Not strictly required, we do show `sam-app` as the prompt during --guided flows. Should we prefix with `sam-app-{}` to give it a feel of a namespace?

Reply:
I like it.